### PR TITLE
master/5.0: bump version and links to v5.0.0 (#5533)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #5533 to release-5.0
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In docs repo:
git pr https://github.com/pingcap/docs/pull/5560
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/docs.git pr/5560:release-5.0-84b484f29c58
```

---

### What is changed, added or deleted? (Required)

Update the code examples and links after 5.0 GA, so we guide users to try a newer version of TiDB.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5773
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@CharLotteiu Any update?

File: maintain-tidb-using-tiup.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ When the cluster is in operation, if you need to modify the parameters of a comp
 
     **Use `.` to represent the hierarchy of the configuration items**.
 
-    For more information on the configuration parameters of components, refer to [TiDB `config.toml.example`](https://github.com/pingcap/tidb/blob/v4.0.0-rc/config/config.toml.example), [TiKV `config.toml.example`](https://github.com/tikv/tikv/blob/v4.0.0-rc/etc/config-template.toml), and [PD `config.toml.example`](https://github.com/pingcap/pd/blob/v4.0.0-rc/conf/config.toml).
+    For more information on the configuration parameters of components, refer to [TiDB `config.toml.example`](https://github.com/pingcap/tidb/blob/master/config/config.toml.example), [TiKV `config.toml.example`](https://github.com/tikv/tikv/blob/master/etc/config-template.toml), and [PD `config.toml.example`](https://github.com/tikv/pd/blob/master/conf/config.toml).
```
Comment (CharLotteiu) [inline maintain-tidb-using-tiup.md:115]:
```suggestion
    For more information on the configuration parameters of components, refer to [TiDB `config.toml.example`](https://github.com/pingcap/tidb/blob/release-5.0/config/config.toml.example), [TiKV `config.toml.example`](https://github.com/tikv/tikv/blob/release-5.0/etc/config-template.toml), and [PD `config.toml.example`](https://github.com/tikv/pd/blob/release-5.0/conf/config.toml).
```

File: maintain-tidb-using-tiup.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ When the cluster is in operation, if you need to modify the parameters of a comp
 
 ### Example
 
-If you want to set the transaction size limit parameter (`txn-total-size-limit` in the [performance](https://github.com/pingcap/tidb/blob/v4.0.0-rc/config/config.toml.example) module) to `1G` in tidb-server, edit the configuration as follows:
+If you want to set the transaction size limit parameter (`txn-total-size-limit` in the [performance](https://github.com/pingcap/tidb/blob/master/config/config.toml.example) module) to `1G` in tidb-server, edit the configuration as follows:
```
Comment (CharLotteiu) [inline maintain-tidb-using-tiup.md:127]:
```suggestion
If you want to set the transaction size limit parameter (`txn-total-size-limit` in the [performance](https://github.com/pingcap/tidb/blob/release-5.0/config/config.toml.example) module) to `1G` in tidb-server, edit the configuration as follows:
```

**Review by CharLotteiu — APPROVED** [review]

**Pull request merged**